charts/rudderstack/templates/statefulset.yaml

```diff
@@ -90,13 +90,13 @@ spec:
           - name: DEST_TRANSFORM_URL
             value: "http://{{ include "transformer.fullname" . }}:{{ .Values.transformer.service.port}}"
           - name: COMPUTE_DB_HOST_IN_K8S
-            value: "true"
+            value: "false"
           - name: POSTGRES_POD_NAME
             value: "{{ .Release.Name }}-{{ .Values.postgresql.nameOverride }}"
           - name: POSTGRES_HEADLESS_SVC
             value: "{{ .Release.Name }}-{{ .Values.postgresql.nameOverride }}-headless"
           - name: JOBS_DB_HOST
-            value: "This is expected to be set by docker entrypoint script"
+            value: "test-rudderstack.cgb7ab6pl842.ap-south-1.rds.amazonaws.com"
```

Comment:
@swayamarora1 You can read these values from `values.yaml` file, rather than hardcoding it here. That will allow you to update it without modifying the charts.

Reply:
See line#101 for reference.